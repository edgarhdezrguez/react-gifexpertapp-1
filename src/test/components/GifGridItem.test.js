import React from 'react'
import { shallow, configure } from 'enzyme';
import '@testing-library/jest-dom'
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import GifGridItem from '../../components/GifGridItem';


configure({ adapter: new Adapter() });
describe('GifGridItem tests', () => {

    const id = "GRSnxyhJnPsaQy9YLn";
    const title = "Dragon Ball GIF by TOEI Animation UK" ;
    const url = "https://media1.giphy.com/media/GRSnxyhJnPsaQy9YLn/giphy.gif?cid=0a7c9746d10r4nkrodijl02nh1q6lbcx2gk4e6vl1h6dfyti&rid=giphy.gif";
    const wrapper = shallow(<GifGridItem id={id} title={title} url={url}/>);
    
    test('should be show the component correctly', () => {

        expect(wrapper).toMatchSnapshot()

    })
    
})
