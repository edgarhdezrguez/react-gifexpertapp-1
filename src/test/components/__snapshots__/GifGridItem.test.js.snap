// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GifGridItem tests should be show the component correctly 1`] = `
<div
  className="card animate__animated animate__fadeIn "
>
  <img
    alt="Dragon Ball GIF by TOEI Animation UK"
    src="https://media1.giphy.com/media/GRSnxyhJnPsaQy9YLn/giphy.gif?cid=0a7c9746d10r4nkrodijl02nh1q6lbcx2gk4e6vl1h6dfyti&rid=giphy.gif"
  />
  <p>
    Dragon Ball GIF by TOEI Animation UK
  </p>
</div>
`;
